lib/expressapp.js

```diff
@@ -27,9 +27,10 @@ ExpressApp.start = function(opts) {
   WalletService.initialize(opts.WalletService);
   var app = express();
   app.use(function(req, res, next) {
-    res.setHeader('Access-Control-Allow-Origin', '*');
+    res.setHeader('Access-Control-Allow-Origin', req.headers.origin);
+    res.setHeader('Access-Control-Allow-Credentials', 'true');
```

Comment:
No, we don't. Thank you for clarifying my doubt. I did a change on `bitcore-wallet-client` to disable withCredentials params on requests. So It 's not necessary to make more changes on `bitcore-wallet-service` than this.

I updated the PR.